testing/trino-product-tests/src/main/java/io/trino/tests/kafka/TestKafkaAvroReadsSmokeTest.java

```diff
@@ -28,6 +28,7 @@
 import io.trino.tempto.fulfillment.table.kafka.ListKafkaDataSource;
```

Comment:
Update commit message to:
```
Allow TestingAvroSchema to resolve SchemaReferences
```

It is not a cleanup to me, but a functional change.